### PR TITLE
Support convTranspose

### PR DESCRIPTION
**Pull request by eguchi1904**
support 2d convTranspose 
https://github.com/onnx/onnx/blob/main/docs/Operators.md#ConvTranspose

### CODE REVIEW

File: runtime/tests/utils.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def check_by_data(expected, result, max_error=1e-4):
 
 def _convert_type(dtype):
     assert not WITHOUT_ONNXRUNTIME
-    return mapping.NP_TYPE_TO_TENSOR_TYPE[dtype]
+    return helper.np_dtype_to_tensor_dtype(dtype)
```
Comment (eguchi1904) [inline runtime/tests/utils.py:84]: using ` mapping.NP_TYPE_TO_TENSOR_TYPE[dtype]` cause following deperecation warning.
```
 DeprecationWarning: `mapping.NP_TYPE_TO_TENSOR_TYPE` is now deprecated and will be removed in a future release.To silence this warning, please use `helper.np_dtype_to_tensor_dtype` instead.
    return mapping.NP_TYPE_TO_TENSOR_TYPE[dtype]
```

File: runtime/README.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ This runtime supports only below operators.
 - ConcatFromSequence
 - Constant
 - ConstantOfShape
+- ConvTranspose
+  - support 2d only
+  - `group` should be 1
+  - `auto_pad` should be `NONE` or `"NOTSET"`
```
Comment (ishiy1993) [inline runtime/README.md:67]:
```suggestion
  - `auto_pad` should be `"NOTSET"` (default value)
```

File: runtime/onnion_runtime/convtranspose.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    kernel_shape: Optional[List[int]]
+    output_shape: Optional[List[int]]
+    output_padding: Optional[List[int]]
+    pads: Optional[List[int]]
+
+    def __init__(self, opset_version: int, **kwargs: Any):
+        self.version = opset_version
+        self.auto_pad = kwargs.get("auto_pad", "NOTSET")
+        self.dilations = kwargs.get("dilations", None)
+        self.group = kwargs.get("group", None)
```
Comment (ishiy1993) [inline runtime/onnion_runtime/convtranspose.py:23]:
```suggestion
        self.group = kwargs.get("group", 1)
```

**Review by ishiy1993 — CHANGES_REQUESTED** [review]
Add tests which use `b`.

**Review by ishiy1993 — APPROVED** [review]
LGTM

**Comment by eguchi1904** [comment]
:pray:

**Pull request merged**